commands/fun/trivia.js

```diff
@@ -0,0 +1,67 @@
+const Social = require(`${process.cwd()}/base/Social.js`);
+const Discord = require("discord.js");
+const snek = require("snekfetch");
+const B64 = require("base-64");
+class Trivia extends Social {
+  constructor(client) {
+    super(client, {
+      name: "trivia",
+      description: "Delivers a quiz with the correct answer awarding 10 points.",
+      category: "Fun",
+      usage: "trivia",
+      extended: "Ready to wrack your brain? ",
+      cooldown: 10,
+      aliases: ["quiz"],
+      botPerms: ["EMBED_LINKS"]
+    });
+  }
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+
+    const inf = await snek.get("https://opentdb.com/api.php?amount=10&category=9&difficulty=medium&type=multiple&encode=base64");
+    const ob = await inf.body;
+    const emb = new Discord.MessageEmbed();
+    var a;
+    var b;
+    var c;
+    var d;
+    var choice;
+    const questions = [];
+
+    ob.results[0].incorrect_answers.forEach(ob => questions.push(B64.decode(ob)));
+    questions.push(B64.decode(ob.results[0].correct_answer));
+    
+    a = await questions.random();
```

Comment:
I assume GuideBot has an `Array.prototype.random()`, but no, it doesn't return a promise. Remove that useless `await` please.